seven-days-to-die.kvp

```diff
@@ -25,7 +25,7 @@ App.LinuxCommandLineArgs=
 App.WindowsCommandLineArgs=
 App.CommandLineArgs=-quit -batchmode -nographics -configfile={{ConfigFile}}.xml {{$FormattedArgs}} -dedicated
 App.AppSettings={}
-App.EnvironmentVariables={"LD_LIBRARY_PATH":"{{$FullBaseDir}}linux64:{{$FullRootDir}}linux64:%LD_LIBRARY_PATH%","SteamAppId":"251570"}
+App.EnvironmentVariables={"DOORSTOP_ENABLE":"TRUE","DOORSTOP_INVOKE_DLL_PATH":"./BepInEx/core/BepInEx.Preloader.dll","DOORSTOP_CORLIB_OVERRIDE_PATH":"./BepInEx/core","LD_LIBRARY_PATH":"./doorstop_libs:{{$FullBaseDir}}linux64:%LD_LIBRARY_PATH%","LD_PRELOAD":"libdoorstop_x64.so","SteamAppId":"251570"}
```

Comment:
Looks like you stomped on the `{{$FullRootDir}}linux64:` portion.

Reply:
Ah yes I messed up and put ```{{$FullBaseDir}}``` instead fixed.